### PR TITLE
refactor hardcoded colors to _globals.ts

### PR DESCRIPTION
**Pull request by 9Syncopation**
add globals.ts color variable connected to the relevant components

### CODE REVIEW

**Comment by 9Syncopation** [comment]
further refactor may be needed for _globals.ts file, some variables holds the same color

File: src/components/molecules/PieChartView.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ interface IProps {
 
 // hardcoded colors, will be changed
 //TODO CHANGE COLORS
-const COLORS = ['#AE0721', '#818386', '#d90000', '#890505', '#6a6a6a'];
+const COLORS = collisionPieChartColors;
```
Comment (yuvalbl) [inline src/components/molecules/PieChartView.tsx:24]: This is a generic pie chart. It should know nothing about "collision"
rename `collisionPieChartColors` to `pieChartColors`

File: src/components/molecules/PieChartView.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export const renderCollisionCustomizedLabel = (props: any, fontSize = '100%', us
   const ey = cy + outerRadius * 0.4 * sin - 20;
 
   const collisionLabelStyle = {
-    color: 'white',
+    color: collisionPieLabelStyleColor,
```
Comment (yuvalbl) [inline src/components/molecules/PieChartView.tsx:37]: same as above - remove the word "collision"

File: src/components/molecules/TableView.tsx [hunk 3]
```diff
@@ -29,7 +29,7 @@ const StyledTableCell = withStyles((theme: Theme) =>
     },
     sizeSmall: {
       padding: '5px 0px',
-      border: '1px solid rgba(0, 0, 0, 0.12)',
+      border: `1px solid ${sizeSmallBorderColor}`,
```
Comment (yuvalbl) [inline src/components/molecules/TableView.tsx:32]: The point of making colors global is to make colors reuseable.
rename`sizeSmallBorderColor` to `borderColor`

**Review by yuvalbl — APPROVED** [review]

**Pull request merged**